### PR TITLE
Add Version KV to telemetry events

### PR DESCRIPTION
**Pull request by cce**
## Summary

This piggybacks off the places where "session" is added to telemetry events, to additionally add a "v" key containing the version string.

## Test Plan

Examine log output from tests.

### CODE REVIEW

File: logging/telemetry.go [hunk 1]
```diff
@@ -264,6 +266,7 @@ func (t *telemetryState) logTelemetry(l logger, message string, details interfac
 	entry := l.entry.WithFields(Fields{
 		"session":      l.GetTelemetrySession(),
 		"instanceName": l.GetInstanceName(),
+		"v":            l.GetTelemetryVersion(),
```
Comment (winder) [inline logging/telemetry.go:269]: Could you use a full string for these?
```suggestion
		"version":            l.GetTelemetryVersion(),
```

File: logging/telemetryFilteredHook.go [hunk 2]
```diff
@@ -82,6 +82,13 @@ func (hook *telemetryFilteredHook) Fire(entry *logrus.Entry) error {
 	} else {
 		newEntry = entry.WithField("session", hook.sessionGUID)
 	}
+
+	// Also add version field, if not already present.
+	if _, has := entry.Data["v"]; has {
+		newEntry = entry
+	} else {
+		newEntry = entry.WithField("v", hook.telemetryConfig.Version)
+	}
```
Comment (winder) [inline logging/telemetryFilteredHook.go:91]:
```suggestion
	// Also add version field, if not already present.
	if _, has := entry.Data["v"]; has {
		newEntry = newEntry
	} else {
		newEntry = newEntry.WithField("v", hook.telemetryConfig.Version)
	}
```

**Review by winder — CHANGES_REQUESTED** [review]
Looks good, just noticed one small bug

File: logging/telemetry.go (hunk 1 above)
Comment (cce) [reply logging/telemetry.go:269]: The concern was that it would use up more bandwidth, so squish it down to the minimum.

**Review by tsachiherman — APPROVED** [review]

**Pull request merged**